staging/src/k8s.io/apimachinery/pkg/watch/mux.go

```diff
@@ -156,26 +160,27 @@ func (m *Broadcaster) WatchWithPrefix(queuedEvents []Event) Interface {
 			w.result <- e
 		}
 	})
+	if w == nil {
+		panic("broadcaster already stopped")
```

Comment:
Add a comment saying this matches previous behavior, but that we're open to reassessing in the future.

Reply:
done